src/lib/integration_tests/tests/platform_orchestrator/provision_subnet_orchestrator.rs

```diff
@@ -87,6 +87,7 @@ fn provision_subnet_orchestrator_canister() {
     let pocket_ic = PocketIcBuilder::new()
         .with_nns_subnet()
         .with_application_subnet()
+        .with_application_subnet()
```

Comment:
repeated line

Reply:
This is needed. We are checking deployment on different subnet.